frontend/views/containers/chatroom/DMMixin.js

```diff
@@ -36,45 +37,69 @@ const DMMixin: Object = {
     },
     createPrivateDM (username: string) {
       // NOTE: username should be valid
-      sbp('gi.actions/mailbox/createDirectMessage', {
-        contractID: this.currentIdentityState.attributes.mailbox,
-        data: {
-          privacyLevel: CHATROOM_PRIVACY_LEVEL.PRIVATE,
-          usernames: [username]
-        }
-      })
+      try {
+        sbp('gi.actions/mailbox/createDirectMessage', {
+          contractID: this.currentIdentityState.attributes.mailbox,
+          data: {
+            privacyLevel: CHATROOM_PRIVACY_LEVEL.PRIVATE,
+            usernames: [username]
+          }
+        })
+      } catch (err) {
+        const errMsg = L('Error during create direct message: {err}', { err: err.message })
```

Comment:
You do not need to reconstruct the error message, it is already correctly generated by the `gi.actions`, simply call `alert` with `err.message`. Same for all of the ones below.

Please also test forcefully throwing these errors from within the `gi.actions` to verify that what the user sees makes sense. For example, if there is HTML included (e.g. a link to the troubleshooting page), then maybe it is better to use `gi.ui/prompt` to display the error instead of `alert` (and make sure it is rendered correctly).

Reply:
Nice catch. I will fix it.

PS: This seems to be related to the #1696.